Assets/Photon Unity Networking/Resources/PhotonServerSettings.asset

```diff
@@ -15,133 +15,17 @@ MonoBehaviour:
   AppID: 
   VoiceAppID: 
   ChatAppID: 
-  HostType: 2
```

Comment:
Should not include this file's changes, as this will break multiplayer functionality with these settings

Reply:
Will remove this thank you.